File: ALODWebUtility.CS/TabNavigator/TabControlBase.cs
using System.Web.UI;

namespace ALODWebUtility.TabNavigation
{
    public abstract class TabControlBase : UserControl
    {
        public abstract bool BackEnabled { get; set; }
        public abstract bool NextEnabled { get; set; }
        public abstract System.Web.UI.WebControls.Button this[ALOD.Core.Domain.Common.NavigatorButtonType type] { get; }
    }
}

File: ALODWebUtility/TabNavigator/TabControlBase.vb
Imports System.Web.UI

Namespace TabNavigation

    Public MustInherit Class TabControlBase
        Inherits UserControl

        Public MustOverride Property BackEnabled() As Boolean
        Public MustOverride Property NextEnabled() As Boolean
        Public MustOverride ReadOnly Property Item(ByVal type As ALOD.Core.Domain.Common.NavigatorButtonType) As System.Web.UI.WebControls.Button
    End Class

End Namespace